### PR TITLE
Add bob.math recipe

### PR DESCRIPTION
**Pull request by 183amir**

### CODE REVIEW

**Comment by 183amir** [comment]
As far as I understand, this is the way to package libraries which build against `BLAS`.
Is this good to get merged?

File: recipes/bob.math/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  fn: bob.math-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.math/bob.math-{{ version }}.zip
+  md5: 0f010af6ce20fe6614570edff94e593f
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+  features:
+    - nomkl
```
Comment (ocefpaf) [inline recipes/bob.math/meta.yaml]: LGTM. But let's wait for @jakirkham or @msarahan here :+1: here before merging.

Comment (jakirkham) [inline recipes/bob.math/meta.yaml]: Thanks for waiting. :smile:

**Comment by jakirkham** [comment]
How exactly to handle BLAS selection correctly is under active discussion. Please see this issue ( https://github.com/conda-forge/staged-recipes/issues/80 ).

**Comment by 183amir** [comment]
Yes, I have seen that. Should we wait until that one gets resolved?

**Comment by jakirkham** [comment]
I was thinking about it. I think I saw you had mentioned somewhere that you were using Linux primarily. Is that true with this package as well?

**Comment by 183amir** [comment]
Yes. Although I would be happy to have mac builds, I cannot really handle them alone. So for now, we are just targeting Linux.

**Comment by 183amir** [comment]
For example, this one (https://github.com/conda-forge/staged-recipes/pull/305) is failing on mac with a segfault and I have no idea how to debug it.

**Comment by jakirkham** [comment]
Yeah, this is fine.

Sorry, I am just thinking about other BLAS issues. I'm just not a big fan of using the Accelerate Framework on Mac for a BLAS/LAPACK. It is buggy, old, and not forksafe (can't use Python's `multiprocessing` library). However, this is the choice Continuum has made with everything else and what you have done here is perfectly consistent. Sorry for the hold up. We may revisit to provide a better BLAS strategy for Mac in the future.

**Comment by jakirkham** [comment]
Thanks @183amir.

**Comment by jakirkham** [comment]
Missed this before. We are linking against NumPy here. Could you please do the following.

1. Fork the feedstock.
2. Create a new branch.
3. Add `numpy x.x` to the `build` and `run` requirements.
4. Add and Commit.
5. Download `conda-smithy` from the `conda-forge` channel into the `root` environment.
6. Run `conda-smithy rerender`.
7. Add and comit.
8. Push and create a PR on the feedstock here.

Please ping me once you have done this. If you have questions, please ask.

**Pull request merged**